app/src/main/java/chat/rocket/android/fragment/server_config/LoginContract.java

```diff
@@ -23,13 +26,13 @@
 
         void showTwoStepAuth();
 
-        void goBack();
+        void goBack(Context ctx);
```

Comment:
We don't need to pass a Context to a `view` implementation since they're supposed to be either an Activity or a Fragment and hence we can get a proper Activity Context from there.